frontend/src/pages/chat/libs/constants/constants.ts

```diff
@@ -0,0 +1,2 @@
+export { MOCK_MESSAGES } from './chat-mock-messages.constant.js';
+export { DEFAULT_INPUT } from './default-input.constant.js';
```

Comment:
define this constants in this file directly, without two additional constant files